WiserTaskScheduler/WiserTaskScheduler/Core/Services/LogService.cs

```diff
@@ -108,8 +113,15 @@ await databaseConnection.ExecuteAsync(@$"INSERT INTO {WiserTableNames.WtsLogs} (
                             // If writing to the database fails log its error.
                             logger.Log(logLevel, $"Failed to write log to database due to exception: ${e}");
                         }
-
                         logger.Log(logLevel, message);
+
+                        // If Slack channel set use Slack .
+                        if (logLevel == logSettings.SlacklogLevel)
+                        {
+                            var slack = scope.ServiceProvider.GetRequiredService<ISlackApiClient>();
+                            var slackChannel = Environment.GetEnvironmentVariable("SlackChannel") ?? slackSettings.SlackChannel;
```

Comment:
Waarom gebruik je hier `Environment.GetEnvironmentVariable`? Die waarde wordt nergens in de code gevuld en de WTS gebruikt ook nergens anders die functie. Graag weghalen en alleen `slackSettings.SlackChannel` gebruiken, tenzij je hier een goede reden voor hebt.